### ISSUE TITLE
YoloV5s weight clarification

### ISSUE THREAD

**Issue by Davidyz** [LilacMeteorObservatory/MetDetPy#3]
Hi, thanks for the amazing work!

Sorry for not using an issue template, but I think this is more of a question about usage rather than a bug/feature request so the templates don't fit.

I noticed that you're using yolov5s model for meteor detection, and I'd like to know whether it's a pre-trained model (from ultralytics etc.) or fine-tuned with your own meteor image data? If it's a pre-trained model, would it be possible to replace it with a larger yolo model and expect potentially improved detection? (less false positives/false negatives)

**Comment by Designerspr** [comment]
Hi Davidyz,

Thank you for your attention to our work!

As you mentioned, our provided YOLOv5s model is fine-tuned with meteor image data. During model training, positive samples are collected from true positive samples (and manually checked false negative samples) of base detector results, so the annotation process doesn't take much time.

Since there is no "meteor" category in the official COCO-pretrained YOLO weights, we created our own [label mapping](https://github.com/LilacMeteorObservatory/MetDetPy/blob/main/config/class_name.txt) for meteor detection scenes. For the same reason, the official YOLO weights cannot be used directly. MetDetPy supports using your own YOLO model (as long as it is converted to ONNX format), but its labels should also be compatible with the above label mapping file. We will provide related guidelines on this.

We have also noticed the limited performance of our provided YOLOv5s model. This issue has been added to our next stage to-do list with high priority. We plan to provide a better fine-tuned YOLOv5s model in the next release version. Additionally, larger-scale models and quantized models are also under development.
